tests/test_telemetry_client.py

```diff
@@ -182,6 +182,17 @@ async def test_handle_telemetry(self):
                 await self.publish_data(tma_data)
                 await self.assert_next_telemetry(topic, desired_sal_data)
 
+    async def test_heartbeat(self):
+        async with self.make_all():
+            if not hasattr(self.remote, "tel_telemetryClientHeartbeat"):
```

Comment:
Can you use `self.telemetry_client.has_heartbeat_topic` instead?

Reply:
I removed the has_heartbeat_topic attribute.